.github/workflows/ci.yml

```diff
@@ -23,8 +23,9 @@ jobs:
           echo "TOOL_NAME=$TOOL_NAME" >> $GITHUB_ENV
           echo "RELEASE=false" >> $GITHUB_ENV
           echo "VERSION=0.0.0.ci0" >> $GITHUB_ENV
+          echo $GITHUB_REF
           echo "TOOL_DIR=$TOOL_NAME" >> $GITHUB_ENV
-          if [[ $GITHUB_REF == refs/tags/v* ]]; then
+          if [[ $GITHUB_REF == refs/tags/v* || "$GITHUB_REF" == *"refs/tags/test-v"* ]]; then
```

Comment:
Syntax should be unified. Either `$GITHUB_REF == refs/tags/v* || "$GITHUB_REF" == refs/tags/test-v*` or `"$GITHUB_REF" == *"refs/tags/v"* || "$GITHUB_REF" == *"refs/tags/test-v"*`.
(I think it should be the former)

Reply:
This one works:
`if [[ "$GITHUB_REF" == *"refs/tags/v"* || "$GITHUB_REF" == *"refs/tags/test-v"* ]]`

Reply:
I'm using if [[ "$GITHUB_REF" == *"refs/tags/v"* || "$GITHUB_REF" == *"refs/tags/test-v"* ]] as @RAm provided me. But anyway, this is not the principal issue.